hat/assets/js/apps/Iaso/domains/instances/components/SpeedDialInstance/speedDialActions.tsx

```diff
@@ -0,0 +1,263 @@
+/* eslint-disable camelcase */
```

Comment:
this file can be move to hooks too

Reply:
I think it makes more sense to keep files that go together next to each other rather than group them by type (especially since these are really specific to this component), but since we do so everywhere else, I'll do it. I just don't think it makes the code base more legible